<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Dag Andersen &lt;dag.andersen@kdemail.net&gt;</author>
 <class>KPlato::TaskProgressPanelBase</class>
 <widget class="QWidget" name="KPlato::TaskProgressPanelBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>427</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Task:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="taskName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="started">
        <property name="text">
         <string>Started:</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="startTime">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Finished:</string>
        </property>
        <property name="buddy">
         <cstring>finishTime</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="finishTime">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="finished">
        <property name="text">
         <string/>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>36</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="editModeWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Edit mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="editmode">
           <item>
            <property name="text">
             <string>Per resource</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Per task</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="ui_resourceFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSpinBox" name="ui_year">
          <property name="minimum">
           <number>1900</number>
          </property>
          <property name="maximum">
           <number>2999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prevWeekBtn">
          <property name="text">
           <string>Prev</string>
          </property>
          <property name="icon">
           <iconset theme="go-previous">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="weekNumber"/>
        </item>
        <item>
         <widget class="QPushButton" name="nextWeekBtn">
          <property name="text">
           <string>Next</string>
          </property>
          <property name="icon">
           <iconset theme="go-next">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="addResourceWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Add resource:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="ui_resourceCombo"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="UsedEffortEditor" name="resourceTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>70</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="ui_entryFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addEntryBtn">
          <property name="text">
           <string>Add Entry</string>
          </property>
          <property name="icon">
           <iconset theme="list-add">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeEntryBtn">
          <property name="text">
           <string>Remove Entry</string>
          </property>
          <property name="icon">
           <iconset theme="list-remove">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CompletionEntryEditor" name="entryTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>KComboBox</header>
  </customwidget>
  <customwidget>
   <class>CompletionEntryEditor</class>
   <extends>QTableView</extends>
   <header>kptusedefforteditor.h</header>
  </customwidget>
  <customwidget>
   <class>UsedEffortEditor</class>
   <extends>QTableView</extends>
   <header>kptusedefforteditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>started</tabstop>
  <tabstop>startTime</tabstop>
  <tabstop>finishTime</tabstop>
  <tabstop>finished</tabstop>
  <tabstop>editmode</tabstop>
  <tabstop>ui_year</tabstop>
  <tabstop>prevWeekBtn</tabstop>
  <tabstop>weekNumber</tabstop>
  <tabstop>nextWeekBtn</tabstop>
  <tabstop>ui_resourceCombo</tabstop>
  <tabstop>addEntryBtn</tabstop>
  <tabstop>removeEntryBtn</tabstop>
  <tabstop>entryTable</tabstop>
  <tabstop>taskName</tabstop>
  <tabstop>resourceTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
